--- /dev/null
+++ ../packages/split/src/PackageToRepositorySplitter.php
@@ -80,7 +80,7 @@
                 $this->symfonyStyle->warning($message);
             }
 
-            $tag = $this->gitManager->getMostRecentTag($rootDirectory);
+            // $tag = $this->gitManager->getMostRecentTag($rootDirectory);
         }
 
         // If branch not set, default to current branch
@@ -87,7 +87,7 @@
         $branch = $branch ?? $this->gitManager->getCurrentBranch();
 
         // If branch doesn't exist on origin, push it
-        if (! $this->gitManager->doesBranchExistOnRemote($branch)) {
+        if (! $this->gitManager->doesBranchExistOnRemote($branch) && $tag === null) {
             $missingBranchMessage = sprintf('Branch "%s" does not exist on origin, pushing it...', $branch);
             $this->symfonyStyle->note($missingBranchMessage);
             $this->symfonyStyle->writeln($this->gitManager->pushBranchToRemoteOrigin($branch));
